client-app/pages/account/addresses.vue

```diff
@@ -13,7 +13,11 @@
             <h2 class="text-gray-800 text-3xl font-bold uppercase">{{ title }}</h2>
 
             <VcButton v-if="!editingMode" class="px-3 uppercase border" size="sm" is-outline @click="openEditMode()">
-              {{ isMobile ? "Add new" : "Add new address" }}
+              {{
+                isMobile
+                  ? $t("pages.account.addresses.add_new_address_mobile_button")
```

Comment:
Maybe better to use modificator as suffix: `add_new_address_button_mobile`

Reply:
maybe mobile_add_new_address_button